src/Migrator.php

```diff
@@ -249,27 +103,24 @@ private function ensureEmptyTargetTable($table, Connection $sourceConnection, Co
     /**
      * Ensure that all table triggers from source are recreated in the destination
      *
-     * @param Connection $sourceConnection Originating DB connection
-     * @param Connection $destConnection   Target DB connection
-     *
      * @return void
      * @throws \RuntimeException If DB type not supported
      * @throws \Doctrine\DBAL\DBALException If target trigger cannot be recreated
      */
-    private function migrateTableTriggers(Connection $sourceConnection, Connection $destConnection)
+    private function migrateTableTriggers(string $setName, TablesList $tableCollection): void
     {
-        foreach ($this->tableMigrations as $table => $sampler) {
+        foreach ($tableCollection->getTables() as $table => $sampler) {
             try {
-                $triggerSql = $this->generateTableTriggerSql($table, $sourceConnection);
+                $triggerSql = $this->generateTableTriggerSql($table, $this->sourceConnection);
                 foreach ($triggerSql as $sql) {
-                    $destConnection->exec($sql);
+                    $this->destConnection->exec($sql);
                 }
                 if (count($triggerSql)) {
-                    $this->getLogger()->info("$this->migrationSetName: Migrated " . count($triggerSql) . " trigger(s) on $table");
+                    $this->logger->info("$setName: Migrated " . count($triggerSql) . " trigger(s) on $table");
                 }
             } catch (\Exception $e) {
-                $this->getLogger()->error(
-                    "$this->migrationSetName: failed to migrate '$table' with '" . $sampler->getName() . "': " . $e->getMessage()
+                $this->logger->error(
+                    "$setName: failed to migrate '$table' with '" . $sampler->getName() . "': " . $e->getMessage()
```

Comment:
Is this error correct here?

function name relates to triggers, but error refers to tables

```suggestion
                    "$setName: failed to migrate '$table' triggers with '" . $sampler->getName() . "': " . $e->getMessage()
```

Reply:
Yeah, looks like you're right, good spot!

Reply:
Suggestion applied in b5003ef